### ISSUE TITLE
Broken after upgrade to 18.1.6

### ISSUE THREAD

**Issue by thomasnilsen** [opnsense/core#2347]
Ny OPNSense is broken after upgrade to 18.1.6

- Access to wan interface from internet works.
- Access to lan interface / dns / web etc works from lan.
- Access to internett from lan does NOT work.

My box have two physical 1gig ports and i use vlan's.

This happened after scheduled automatic update at 05:00 yesterday.

Firewall log does not show any entries after update time.

best regards
Thomas

**Comment by fichtner** [comment]
> Access to internett from lan does NOT work.

This is not a lot to go on, unfortunately.

**Comment by thomasnilsen** [comment]
@fichtner What can I do to enlighten you? Tell me what logs etc I can provide.

**Comment by fichtner** [comment]
Can you ping 8.8.8.8 from LAN? Can you ping google.com from LAN? Maybe your IPv6 bogons don't load, increase firewall: settings: advanced: table entries to "500000".

**Comment by thomasnilsen** [comment]
Can ping local lan interface on opnsense from computer.
Can not ping wan interface on opnsense from computer.
Can not ping 8.8.8.8. or anything else from computer.

Can ping anything from opnsense.

dig google.com works from computer (since dns is opnsense lan if and lan<->lan works).

Increased table does not help.

**Comment by fichtner** [comment]
Check System: Log Files: General for errors, but it looks like an operational issue judging by the general lack of an indicator why your LAN would suddenly fail. If you have multi-wan configured, check that. It's almost impossible to help from here.

**Comment by thomasnilsen** [comment]
@fichtner Definetly something wrong with pf. My general log shows SSH login attempts from various sources. Should not be possible. 22 is opened only from 2 ip's.

Also shows the "[ There were error(s) loading the rules: no IP address found for <!DOCTYPE]" message..

**Comment by fichtner** [comment]
Aha, an error message!

Run this, all back to normal hopefully...

    # /usr/local/etc/rc.update_bogons


Cheers,
Franco

**Comment by thomasnilsen** [comment]
@fichtner 
Found problem;

more /usr/local/etc/bogonsv6 shows a html file.. Looks like the bogon download may have failed and downloaded the HTML file.

`



<html lang="en">

<head>

<meta charset="utf-8">
<title>Team Cymru</title>
<meta name="description" content="We exist to enable organizations to efficiently identify and eradicate threats within their network by providing unique and easily assessable insight that saves and improves human lives.">
<meta name="author" content="Team Cymru">




<script type="text/javascript">
  var _paq = _paq || [];
  _paq.push(["trackPageView"]);
  _paq.push(["enableLinkTracking"]);

  (function() {
    var u="https://piwik.cymru.com/";
    _paq.push(["setTrackerUrl", u+"piwik.php"]);
    _paq.push(["setSiteId", "1"]);
    var d=document, g=d.createElement("script"), s=d.getElementsByTagName("script")[0]; g.type="text/javascript"; g.defer=true; g.async=true; g.src=u+"piwik.js"; s.parentNode.insertBefore(g,s);
  })();
</script>


<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">


<link type="text/css" rel="stylesheet" href="bootstrap/css/bootstrap.css">

<link rel="stylesheet" type="text/css" href="js-plugin/animation-framework/animate.css" />

`

**Comment by fichtner** [comment]
Yes, that was reported over a week ago and subsequently fixed. it must have cached the bad version for you. The command should reload it. :)
